assert-camunda-git-emails/check.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -eu
+
+# This variable has to contain a `grep` expression
+HARDCODED_ALLOWED_EMAILS_LIST="@camunda.com\|@users.noreply.github.com\|noreply@github.com"
```

Comment:
maybe add the regex nature to the naming? e.g.
```suggestion
HARDCODED_ALLOWED_EMAILS_REGEX="@camunda.com\|@users.noreply.github.com\|noreply@github.com"
```

Reply:
Good point!